src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,45 +1,37 @@
 package edu.kis.vh.nursery;
 
-public class DefaultCountingOutRhymer {
-
-    private static final int ARRAY_SIZE = 12;
-
-    private static final int CAPACITY = 11;
+import edu.kis.vh.nursery.dataStructures.IntArraySTack;
 
-    private static final int EMPTY_RHYMER_INDICATOR = -1;
-
-    private static final int DEFAULT = -1;
+public class DefaultCountingOutRhymer {
+    public IntArraySTack arraySTack = new IntArraySTack();
```

Comment:
The goal of the lab was to make using IntLinkedList and IntArrayStack interchangeable.